src/common/components/clusterWizard/ClusterWizardStepValidationsAlert.tsx

```diff
@@ -72,20 +89,29 @@ const ClusterWizardStepValidationsAlert = <ClusterWizardStepsType extends string
         <AlertGroup>
           {children}
           <Alert variant={AlertVariant.warning} title="Cluster is not ready yet." isInline>
-            {!!failedClusterValidations.length && (
-              <Flex spaceItems={{ default: 'spaceItemsSm' }} direction={{ default: 'column' }}>
-                <FlexItem>{t('ai:The following requirements must be met:')}</FlexItem>
+            <Flex spaceItems={{ default: 'spaceItemsSm' }} direction={{ default: 'column' }}>
+              {!!failedClusterValidations.length && (
+                <>
+                  <FlexItem>{t('ai:The following requirements must be met:')}</FlexItem>
+                  <FlexItem>
+                    <List>
+                      {(failedClusterValidations.filter(Boolean) as Validation[]).map(
+                        (validation) => (
+                          <ListItem key={validation.id}>{validation.message}</ListItem>
+                        ),
+                      )}
+                    </List>
+                  </FlexItem>
+                </>
+              )}
+              {!!failedHostValidations.length && (
```

Comment:
This doesn't scale well if we need to adapt a different message.
Let's try to make it generic, still iterating over `failedHostValidations`, and printing all its messages.

Then we adapt specific validationIds to specific messages. It can still be done in this component.